lfs/client.go

```diff
@@ -795,3 +820,12 @@ func setErrorHeaderContext(err error, prefix string, head http.Header) {
 		}
 	}
 }
+
+func PrintError(err error, format string, args ...interface{}) {
+	line := format
+	if len(args) > 0 {
+		line = fmt.Sprintf(format, args...)
+	}
+	fmt.Fprintln(ErrorWriter, line)
+	fmt.Fprintln(ErrorWriter, err.Error())
+}
```

Comment:
I understand. My reason for doing this was thinking that the headers are not critical to the operation, so the operation shouldn't fail if any problems come up.